From: Brian May <bam@debian.org>
Date: Wed, 26 Apr 2017 06:29:23 +1000
Subject: No build string

Remove hostname and build time from version as they make the build
unreproducible

Author: Jelmer Vernooij <jelmer@debian.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 838f94f..3425bf8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -657,7 +657,7 @@ cat > include/newversion.h.in <<EOF
 #ifndef VERSION_HIDDEN
 #define VERSION_HIDDEN
 #endif
-VERSION_HIDDEN const char *heimdal_long_version = "@([#])\$Version: $PACKAGE_STRING by @USER@ on @HOST@ ($host) @DATE@ \$";
+VERSION_HIDDEN const char *heimdal_long_version = "@([#])\$Version: $PACKAGE_STRING \$";
 VERSION_HIDDEN const char *heimdal_version = "AC_PACKAGE_STRING";
 EOF
 
